shared/types/user.ts

```diff
@@ -21,8 +21,6 @@ export const UserBase = z.object({
       payment: z.boolean().optional(),
       children: z.boolean().optional(),
       postmanSms: z.boolean().optional(),
-      // TODO: (MRF-email-notif) Remove betaFlag when MRF email notifications is out of beta
-      mrfEmailNotifications: z.boolean().optional(),
```

Comment:
Can retain this betaflag. This is to prevent accidental reuse of prior flags. I.e., we might have another feature that uses the same flag name of `mrfEmailNotification` which is not known to the developer cause it no longer exist in the records, but exist in `User` collection.

Reply:
retained in [d3a5fa7](https://github.com/opengovsg/FormSG/pull/7920/commits/d3a5fa713c5e9a821264a9fcc58718ae497926b0)